src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -185,7 +188,7 @@ public FormValidation doCheckUrl(@AncestorInPath Item item,
             return FormValidation.ok();
         }
 
-        private static StandardCredentials lookupCredentials(Item project, String credentialId, String uri) {
+        private static StandardCredentials lookupCredentials(@CheckForNull Item project, String credentialId, String uri) {
             return (credentialId == null) ? null : CredentialsMatchers.firstOrNull(
                         CredentialsProvider.lookupCredentials(StandardCredentials.class, project, ACL.SYSTEM,
```

Comment:
`lookupCredentials` already accepted null. Perhaps you meant to comment on a different line?